src/tools/convert/gromacs.py

```diff
@@ -1065,6 +1072,55 @@ def setLennardJonesInteractions(system, defaults, atomtypeparams, verletlist, cu
     system.addInteraction(interaction)
     return interaction
 
+def setLennardJonesInteractionsTI(system, defaults, atomtypeparams, verletlist, cutoff, epsilonB, sigmaSC, alphaSC, powerSC, lambdaTI, pidlist, annihilate=True, hadress=False, adress=False, ftpl=None):
+    """ Set lennard jones interactions which were read from gromacs based on the atomypes"""
+    if (hadress and adress):
+      print "Error! In gromacs.setLennardJonesInteractionsTI, you cannot use adress and hadress at the same time"
+      return
+    if (hadress):
+        #interaction=espressopp.interaction.VerletListHadressLennardJonesSoftcoreTI(verletlist, ftpl)
```

Comment:
@acfogarty, adding a tag on the commented code "For Furhter Functionality" or tagging it with "dev further functionality" will be also useful. I found many former comments in the code with "TODO..." which was not self-explanatory